test/quantization/test_numeric_suite_fx.py

```diff
@@ -142,3 +144,96 @@ def compare_and_validate_results(float_model, q_model):
         q_model = convert_fx(prepared_model)
 
         compare_and_validate_results(backup_prepared_model, q_model)
+
+    @skipIfNoFBGEMM
+    def test_compare_model_stub_conv_static_fx(self):
+        r"""Compare the output of static quantized conv layer and its float shadow module"""
+
+        def calibrate(model, calib_data):
```

Comment:
Seems this function is the same for all tests, could you move it out so that you don't need to define it in each test?

Reply:
Or use test_only_eval_fn as in your other PR?